components/attention/w-attention.vue

```diff
@@ -1,84 +1,123 @@
 <script setup>
-  import { watch, computed, ref, onMounted, nextTick } from 'vue'
-  import { attention as ccAttention } from '@warp-ds/css/component-classes'
-  import { computePosition, flip, offset, shift, arrow } from '@floating-ui/dom'
+import { watch, computed, ref, onMounted, nextTick } from "vue";
```

Comment:
We really need to have some tool for code style/standards in place. 😅  
We use single quotes... and I'm not really sure about the indentation? 🤔  But we should not really have to think about these kind of things. 😅  I guess there is a task lying around somewhere to fix this...? 🤷 😄

Reply:
But I can also adjust my prettier-settings, just need to know which code style/standards we have. I can change easily to single qoutes, but what kind of kindentation do we have? 😄

Reply:
In Vue I think we only have indentation for template, and not for script and style (if any). So I guess your change about the indentation was correct. 😄 It is a bit confusing in Vue due to files having being created at different times with different code standards. 😅 I think I would like to do a quick run through of the whole repo and align things... 🙂 So you could just leave it as it is now I guess. 👍